documentation/src/main/wiki/KnotxDeployment.md

```diff
@@ -21,23 +21,22 @@ They will be taken from **knotx-standalone-X.Y.Z-fat.jar**.
 from the [latest release](https://github.com/Cognifide/knotx/releases) as a reference. In this example,
 created file is named `knotx-starter.json` and is placed in `KNOTX_HOME`.
 
-- Create your own [Logback logger configuration](http://logback.qos.ch/manual/configuration.html)
-(any location on the host) basing e.g. on `logback.xml`
-from the [latest release](https://github.com/Cognifide/knotx/releases).
+- Create your own logger configuration. See [[Knot.x Logging|Logging]] on how to do it.
 
 At this step `KNOTX_HOME` should contain:
 ```
 - app
-    - knotx-standalone-X.Y.Z-fat.jar
-    - custom-modules.jar
-- knotx-starter.json
-- logback.xml
+  - custom-modules.jar
+- config
+  - knotx-starter.json
+  - logback.xml
+- knotx-standalone-X.Y.Z-fat.jar
 ```
 
 To start Knot.x with custom modules, use following command
 
 ```
-java -Dlogback.configurationFile=logback.xml -cp "app/*" io.knotx.launcher.LogbackLauncher -conf knotx-starter.json
+java -Dlogback.configurationFile=config/logback.xml -jar knotx-standalone-X.Y.Z-fat.jar -conf config/knotx-starter.json -cp "app/*" 
```

Comment:
it's optimization that will prevent overwriting knotx classes if any jar inside app/ is bundled with other version of knotx.
It's because the "-cp" parameter is the vertx parameter that creates an isolated classloader